src/Symfony/Component/HttpKernel/Log/Logger.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Log;
+
+use Psr\Log\AbstractLogger;
+use Psr\Log\InvalidArgumentException;
+use Psr\Log\LogLevel;
+
+/**
+ * Minimalist PSR-3 logger designed to write in stderr or any other stream.
+ *
+ * @experimental
```

Comment:
We are only using `@experimental` when we are very unsure about a feature. That's not the case here. So `@internal` seems better to me. Or nothing at all :)

Reply:
Ok, I think `@internal` is better for now. We can always  remove this annotation in the future.